src/GrammarReader.hs

```diff
@@ -24,10 +24,10 @@ import qualified System.Environment as SE
 
 import GrammarType
 import CFG2TM
+import ParsingHelpers
+import TuringMachineWriter
```

Comment:
Why GrammarREader requires TMWriter?

Reply:
As I said, for a demonstration purpose I put `tm2tms` (which is in the `TMWriter`) to the `convertGrammar2TM` function.

Reply:
Ah, I see. Ok.